src/store.js

```diff
@@ -1,94 +1,65 @@
 const path = require('path')
 const fs = require('fs')
+const homedir = require('os').homedir()
 
-const global = path.join(__dirname, '..', 'alia.json')
+const defaultConfig = require('./defaultConfig')
+let config = defaultConfig
 
-const separator = `~`
+const configPath = path.join(homedir, '.alia.json')
 
-module.exports.add = function(args, cb) {
-  if (!args.includes(separator)) {
-    return cb(`
-      Invalid Input, missing separator: ~
-    `)
-  }
-
-  const [key, cmd] = args.map(arg => arg).join(' ').split(separator).map(arg => arg.trim())
+function writeConfig() {
+  fs.writeFileSync(configPath, JSON.stringify(config, null, 2))
+}
 
-  readStore((err, alia) => {
-    if (err) {
-      return cb(err)
-    }
+try {
+  // Read config
+  config = require(configPath)
```

Comment:
could use fs.stat instead?

Reply:
`fs.stat` to check the file exists before trying to read? the thingy I wrote effectively does the check and the read in one